hfc/fabric_network/wallet.py

```diff
@@ -28,6 +30,18 @@ def remove(self, enrollment_id):
         if dirpath.exists() and dirpath.is_dir():
             shutil.rmtree(dirpath)
 
+    def get_user(self, enrollment_id, org, msp_id, state_store=None):
+        """ Returns an instance of a user whose identity
+            is stored in the FileSystemWallet
+        """
+        if not self.exists(enrollment_id):
+            raise AttributeError('"user" does not exist')
+        state_store = FileKeyValueStore(self._path)
+        cert_path = self._path + '/' + enrollment_id + '/' + 'private_sk'
+        key_path = self._path + '/' + enrollment_id + '/' + 'enrollmentCert.pem'
```

Comment:
these two paths are just hard-coded. It seems to me they should be parameters

Reply:
**'enrollmet_id'** is a parameter which is taken is the input.